src/app/work-item/work-item.service.ts

```diff
@@ -222,10 +391,21 @@ export class WorkItemService {
         let deletedItemIndex = this.workItems.findIndex((item) => item.id == workItem.id);
         // removing deleted item from the local list
         this.workItems.splice(deletedItemIndex, 1);
+        // Re build the workItem ID-Index map
+        this.buildWorkItemIdIndexMap();
       })
       .catch(this.handleError);
   }
 
+   /**
+   * Usage: This method create a new item 
```

Comment:
this just hurts my sensibilities, sorry but could please align the asterisk in the comment block